src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -254,6 +268,15 @@ private void styleFetchedListView(CheckListView<CitationRelationItem> listView)
                         vContainer.getChildren().addLast(openWeb);
                     }
 
+                    Button showEntrySource = IconTheme.JabRefIcons.SOURCE.asButton();
+                    showEntrySource.setTooltip(new Tooltip(Localization.lang("Show %0 source", databaseContext.getMode().getFormattedName())));
+                    showEntrySource.setOnMouseClicked(event -> {
+                        this.entryEditor = createEntryEditor();
```

Comment:
Why do you need a full entry editor? I think, you can just show the source without full editing capabilities.